### DISCUSSION TITLE
How do I tell Nuke not to ask for input?

### DISCUSSION THREAD

**Discussion by voroninp** [nuke-build/nuke#1010] [Q&A] [answered]
I am running nuke build inside docker, and build fails because of the warning:

> #10 80.55 11:38:25 [WRN] Configuration files for GitHubActions (continuous) have changed.
#10 80.58 Press any key to continue ...
#10 80.60 Cannot read keys when either application does not have a console or when console input has been redirected. Try Console.Read.

1. How do I make nuke not ask for the keypress?
2. Can I prevent generation of gihub actions workflow files with some CLI arg?

**Comment by matkoch** [comment, marked as answer]
`AutoGenerate = false` on the attribute

**Reply by voroninp** [reply]
Thanks. And what arg should be used to generate it? Or do I need to define a dedicated target for workflow generation?

**Reply by matkoch** [reply]
The command is written in the workflow file. I just don't understand why you'd disable it. It's only generated when something has changed, as the warning says.
